### ISSUE TITLE
PADDING is hardcoded in data/__init__.py

### ISSUE THREAD

**Issue by plin1112** [aiqm/torchani#458]
PADDING is hardcoded in data/__init__.py make it difficult to use other properties for training without making change to the code.

PADDING = {
    'species': -1,
    'coordinates': 0.0,
    'forces': 0.0,
    'energies': 0.0
}

**Comment by IgnacioJPickering** [comment]
It is kind of strange to have PADDING be global but I guess the intent is to have it be an "option" of the data module, kind of like a static variable. You can add your own properties by extending padding by doing this:

```
torch.data.PADDING({'new_property' : new_value})
torchani.data.load(dspath) .... 
```

Maybe we could add this to the documentation

**Comment by IgnacioJPickering** [comment]
Hey @plin1112 this was addressed in pr #489 so now you should be able to edit the padding dict more  comfortrably, check out the docs.
